contracts/marketplace/MarketplaceInitFacet.sol

```diff
@@ -61,5 +64,10 @@ contract MarketplaceInitFacet is AMarketplaceFacet {
 
     // Initialize automatons storage.
     LibHasAutomatons.data().version = LibHasAutomatons.STORAGE_VERSION;
+
+    // ------------------------------------- //
+
+    // Initialize forwarder storage.
+    LibHasForwarder.data().forwarderAddress = params.trustedForwarder;
```

Comment:
I don't know how this will work in production.
Adding to the `initialize` function will work locally or when deploying the Smart Contract from scratch...
But in production you can't really call `initialize` again.
Maybe what you could do is add a function to the `InitFacet` such as `migratev1tov2` and add the code there?